.github/workflows/publish-java.yml

```
@@ -0,0 +1,41 @@
+name: Publish package to Maven Central Repository
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
```

When I test this regex on online regex test site, it doesn't include v1.0, v0.1.
To include vx.x, you may use `v[0-9]+(.[0-9]+)?.[0-9]+(-rc[0-9]+)?`.
```suggestion
      - "v[0-9]+(.[0-9]+)?.[0-9]+(-rc[0-9]+)?" # Push events to matching v*, i.e. v1.0, v20.15.10, v1.0-rc1, v20.15.10-rc5
```

---

Since we use `semantic versioning`, I think versioning like `v1.0` would not be happen.
How do you think about this? @zemyblue

---

Right, as we use semantic versioning, `v1.0` is invalid.

---

I think, it's better to correct comments only by removing `v1.0.`

---

I got it